### PR TITLE
Mobile Style Polish

### PR DESCRIPTION
**Pull request by xtine**
For eregs/notice-and-comment#356

General
- set top padding to 30px between header and content
- type hierarchies
- header mobile menu (hamburger) vertically aligned - this affects general eregs

Intro
- Hide days remaining

Tabs and Drawer
- Read/Write tab font size
- Set 15px Read/Write tab spacing in tablet

Read Mode
- Hide "write a comment" hover links
- "Write a comment" links darker blue
- Footer says previous/next and has top border

Write Mode
- Hide excerpt
- Tighten spacing in comment upload attachment and save buttons, and made them full width for visibility

screen shot 2016-06-08 at 10 02 34 pm
screen shot 2016-06-08 at 10 00 30 pm
screen shot 2016-06-08 at 10 00 54 pm
screen shot 2016-06-08 at 10 08 35 pm
screen shot 2016-06-08 at 10 01 10 pm
screen shot 2016-06-08 at 10 01 32 pm

### CODE REVIEW

File: regulations/static/regulations/css/less/module/comment-media-queries.less [hunk 1]
```diff
@@ -61,11 +61,27 @@ comment-media-queries.less contains media query styles for Notice and Comment
...
     display: none;
     padding: 0;
   }
 
+  .preamble-header {
+
+    .read-proposal,
+    .write-comment {
+      padding: 0 15px;
+      width: auto !important;
```
Comment (cmc333333) [inline regulations/static/regulations/css/less/module/comment-media-queries.less]: Small flag around the use of `!important` -- sure there's not a better solution?

**Comment by cmc333333** [comment]
Looks good to me locally, but @donjo would you mind reviewing? I don't know all of the relevant things

**Comment by donjo** [comment]
Looks pretty good!

I think if the mobile "write a comment" links are going to turn darker blue for mobile they should change that color at the resolution where they go from being on the side to being in-line since at tablet resolutions users are unlikely to have a mouse  to hover over them (which I imagine was the idea behind making them darker blue for mobile)

**Comment by xtine** [comment]
@donjo: just updated so tablet size "write a comment" links are darker blue now.

**Comment by xtine** [comment]
@cmc333333: the width thing on `.read-proposal, .write-comment` stumped me, actually. The width is only being set in `comment.less:44` so the media query on `.comment-media-queries.less:81` _should_ override but isn't without !important on it...

**Comment by cmc333333** [comment]
:+1: rockin

**Pull request merged**